### PR TITLE
Next tranche in ongoing development of TSP API. 

### PR DESCRIPTION
**Pull request by ntwyman**
## Description

Added blackouts to the model and fleshed out more of the Shipments and TSP endpoints.

Alphabetized the definitions and paths sections.

## Reviewer Notes

Make sure I haven't broken anything in active use. I want to keep landing this so a) folks can see what's going on & b) I don't want it to diverge to much from Master

## Verification Steps

* [*] All tests pass.
* [n/a] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely) have been satisfied.
* [n/a] There are no [aXe](https://www.deque.com/products/aXe/) warnings for UI.
* [n/a] This works in IE.

### CODE REVIEW

File: pkg/handlers/shipments.go [hunk 1]
```diff
@@ -46,3 +47,47 @@ func (h IndexShipmentsHandler) Handle(p shipmentop.IndexShipmentsParams) middlew
...
+
+// Handle accepts the shipment - checks that currently logged in user is authorized to act for the TSP assigned the shipment
+func (h AcceptShipmentHandler) Handle(p apioperations.AcceptShipmentParams) middleware.Responder {
+	return middleware.NotImplemented("operation .acceptShipment has not yet been implemented")
+}
+
+// RefuseShipmentHandler allows a TSP to accept a particular shipment
+type RefuseShipmentHandler HandlerContext
+
+// Handle accepts the shipment - checks that currently logged in user is authorized to act for the TSP assigned the shipment
```
Comment (tinyels) [inline pkg/handlers/shipments.go]: s/accepts/refuses

**Comment by breanneboland** [comment]
👀 I took a look and will keep watching as the parts that touch blackout dates, in particular, take shape.

File: swagger/api.yaml [hunk 2]
```diff
@@ -14,69 +14,120 @@ consumes:
...
+        404:
+          description: shipment UUID not found in system
+  /shipments/{shipment_UUID}/accept:
     post:
       summary: Accepts an awarded shipment
       description: Accepts a shipment awarded to a TSP
       operationId: acceptShipment
       parameters:
-        - name: shipmentUUID
+        - name: shipment_UUID
```
Comment (macrael) [inline swagger/api.yaml]: I think `shipment_uuid` would be more consistent.

File: pkg/handlers/shipments.go [hunk 3]
```diff
@@ -46,3 +47,47 @@ func (h IndexShipmentsHandler) Handle(p shipmentop.IndexShipmentsParams) middlew
...
+
+// AcceptShipmentHandler allows a TSP to accept a particular shipment
+type AcceptShipmentHandler HandlerContext
+
+// Handle accepts the shipment - checks that currently logged in user is authorized to act for the TSP assigned the shipment
+func (h AcceptShipmentHandler) Handle(p apioperations.AcceptShipmentParams) middleware.Responder {
+	return middleware.NotImplemented("operation .acceptShipment has not yet been implemented")
+}
+
+// RefuseShipmentHandler allows a TSP to accept a particular shipment
```
Comment (breanneboland) [inline pkg/handlers/shipments.go]: Refuse a particular shipment?

**Pull request merged**